docs/_dg/Implementation.md

```diff
@@ -46,3 +46,19 @@ Currently, sorting is performed ascendingly. A future implementation will allow
 There is a `SortedList` obtained from an immutable item list. When the `SortCommand` is executed to sort the list of items, one or more comparators are chosen depending on the provided sorting criteria. the comparators are combined into one comparator. The `SortedList`  are sorted in the specified order of the Comparators. The UI tracks changes to the `SortedList` and displays the updated sorted item list.
 
 Notably, the `SortedList` wraps around a `FilteredList`, which wraps around an immutable `ObservableList`.
+
+#### Receiving help for a command
+The `help` feature provides a user with instructions of how to use certain commands.
+
+#### How the `help` command works
+1. The user specifies a specific command that they need help with. This is done using the word needed to execute a particular command in FoodRem.
+2. If this is not provided, a general help message will be shown.
+3. The provided command is cross-referenced with all available commands in FoodRem. If the command that the user needs help with does not exist, an error would be thrown. This informs the user that the command does not exist. A general help is also shown to the user.
+4. The help is shown in a new window that will open upon successful execution of the command.
```

Comment:
Let's follow the module's markdown style guide and use generic list item numbers.

```suggestion
1. The user specifies a specific command that they need help with. This is done using the word needed to execute a particular command in FoodRem.
1. If this is not provided, a general help message will be shown.
1. The provided command is cross-referenced with all available commands in FoodRem. If the command that the user needs help with does not exist, an error would be thrown. This informs the user that the command does not exist. A general help is also shown to the user.
1. The help is shown in a new window that will open upon successful execution of the command.
```